custom_components/tuya_local/devices/ogacfo_ultrasonic_floor_humidifier.yaml

```diff
@@ -0,0 +1,108 @@
+name: Humidifier
+products:
+  - id: 4wilpf0amoade7i3
+    manufacturer: OGACFO
+    model: LFHM055
+entities:
+  - entity: humidifier
+    class: humidifier
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+        mapping:
+          - dps_val: true
+            icon: "mdi:air-humidifier"
+          - dps_val: false
+            icon: "mdi:air-humidifier-off"
+      - id: 101
+        name: humidity
+        type: integer
+        range:
+          min: 30
+          max: 80
+        mapping:
+          - step: 5
+      - id: 14
+        name: current_humidity
+        type: integer
+      - id: 23
+        type: string
+        name: mode
+        mapping:
+          - dps_val: level_1
+            value: eco
+          - dps_val: level_2
+            value: normal
+          - dps_val: level_3
+            value: boost
+  - entity: switch
+    translation_key: uv_sterilization
+    category: config
+    dps:
+      - id: 5
+        name: switch
+        type: boolean
+  - entity: switch
+    translation_key: sleep
+    category: config
+    dps:
+      - id: 16
+        type: boolean
+        name: switch
+  - entity: binary_sensor
+    translation_key: tank_empty
+    category: diagnostic
+    dps:
+      - id: 22
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+            icon: "mdi:water"
```

Comment:
Don't override the icons here. The tank_empty translation_key should already do it for you.